examples/zot.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=OCI Distribution Registry
+Documentation=https://github.com/anuvu/zot
+After=network.target auditd.service local-fs.target
+
+[Service]
+Type=simple
+ExecStart=/usr/bin/zot serve /etc/zot/config.json
```

Comment:
I wonder if this should just be `zot`?

Reply:
Would the user of this to tweak, but basically not to be left to the devices of PATH env var.